bundles/org.openhab.binding.mercedesme/src/main/resources/OH-INF/thing/eco-channel-types.xml

```diff
@@ -4,25 +4,25 @@
 	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
 	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
 	<channel-type id="accel">
-		<item-type>Number:Dimensionless</item-type>
+		<item-type unitHint="%">Number:Dimensionless</item-type>
 		<label>Acceleration Score</label>
 		<description>Score for smooth acceleration</description>
 		<state readOnly="true"/>
 	</channel-type>
 	<channel-type id="coasting">
-		<item-type>Number:Dimensionless</item-type>
+		<item-type unitHint="%">Number:Dimensionless</item-type>
 		<label>Coasting Score</label>
 		<description>Score for driving without braking</description>
 		<state readOnly="true"/>
 	</channel-type>
 	<channel-type id="constant">
-		<item-type>Number:Dimensionless</item-type>
+		<item-type unitHint="%">Number:Dimensionless</item-type>
 		<label>Constant Score</label>
 		<description>Score for driving with constant speed</description>
 		<state readOnly="true"/>
 	</channel-type>
 	<channel-type id="bonus">
-		<item-type>Number:Length</item-type>
+		<item-type unitHint="km">Number:Length</item-type>
```

Comment:
I don't remember the correct abbreviation for miles (mi?), but you can provide a comma-separated list here. 
This way it will default the unit corresponding to the configured measurement system (metric or imperial).

See for example:

https://github.com/openhab/openhab-addons/blob/80d64ccd06df8827967384020ed53c997ef1fa07/bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/channeltypes.xml#L254-L264

Reply:
Good hint, I'll do so.

Reply:
Imperial unit hints added. 
mi for miles, gal for volume and psi for pressure